portmap/core/views.py

```diff
@@ -216,3 +219,54 @@ def debug_list_articles(request):
 
 def debug_help_dev(request):
     return TemplateResponse(request, "core/debug_index.html")
+
+class CorrectMimeTypeFeed(DefaultFeed):
+    content_type = 'application/xml; charset=utf-8'
+
+class RssFeed(Feed):
+    feed_type = CorrectMimeTypeFeed
+    title_template = "feed/article_title.html"
+    description_template = "feed/article_description.html"
+    title = "RSS Feed"
+    link = "https://portmap.dtinit.org/articles_feed"
+    description = "Articles from Portability Articles repo"
+
+    def items(self):
+        github_articles = "https://api.github.com/repos/dtinit/portability-articles/contents/articles"
+        response = requests.get(github_articles)
```

Comment:
I'm surprised we'd be getting the articles from Github when they're already in our portmap database?  Unless there's a reason I don't know about, I think it would be a lot better to get them from our database.

Reply:
Working on it. You're right